### PR TITLE
WT-8650 Fix malformed comments in all source files

### PR DESCRIPTION
**Pull request by jiechenbo**
The previous ticket [WT-8274](https://jira.mongodb.org/browse/WT-8274), did not take care of malformed comments and instead added a TODO comment on top of the each malformed comment. This tickets aims to fix through deleting the duplicate TODO comments and fixing the malformed comments.

### CODE REVIEW

File: test/bloom/test_bloom.c [hunk 1]
```diff
@@ -226,8 +226,9 @@ cleanup(void)
...
 
 /*
- * Create and keep all the strings used to populate the bloom filter, so that we can do validation
- * with the same set of entries.
+ * populate_entries --
+ *     Create and keep all the strings used to populate the bloom filter, so that we can do
+ *     validation with the same set of entries.
  */
 /*
  * populate_entries --
```
Comment (etienneptl) [inline test/bloom/test_bloom.c:234]: This one should be removed

File: test/csuite/rwlock/main.c [hunk 2]
```diff
@@ -92,7 +92,8 @@ main(int argc, char *argv[])
 }
 
 /*
- * Acquire a rwlock, every Nth operation, acquire exclusive.
+ * thread_rwlock --
+ *     Acquire a rwlock, every Nth operation, acquire exclusive.
  */
 /*
  * thread_rwlock --
```
Comment (etienneptl) [inline test/csuite/rwlock/main.c:99]: Should be removed too

File: test/csuite/wt2999_join_extractor/main.c [hunk 3]
```diff
@@ -28,15 +28,17 @@
...
- * join is created using the two indices, and iterated.
+ *     pair of custom extractors that pull the first and second 32-bit integers from the blob. A
+ *     simple join is created using the two indices, and iterated.
  *
  * Failure mode: When a custom extractor is used with cursor joins, there are memory leaks at the
- * point where the extractor sets the key.
+ *     point where the extractor sets the key.
  */
+
 /*
```
Comment (etienneptl) [inline test/csuite/wt2999_join_extractor/main.c:42]: Need to be removed

File: src/utilities/util_load_json.c [hunk 4]
```diff
@@ -559,24 +559,3 @@ json_skip(WT_SESSION *session, JSON_INPUT_STATE *ins, const char **matches)
  * util_load_json --
  *     Load from the JSON format produced by 'wt dump -j'.
  */
-/*
```
Comment (etienneptl) [inline src/utilities/util_load_json.c:562]: Was the intended to be removed? If yes should we remove the header of the function as well?

File: test/csuite/wt3338_partial_update/main.c [hunk 5]
```diff
@@ -138,26 +138,18 @@ compare(WT_ITEM *orig, WT_ITEM *local, WT_ITEM *library)
...
- *	inputs into the calculate-modify API to generate a set of modify
- *	vectors. Apply this generated vector to the initial value using the
- *	modify apply API to obtain a final value. The final value generated
- *	should match the modified value that was used as input to the
- *	calculate-modify API.
- */
 /*
  * modify_run --
- *     TODO: Add a comment describing this function.
+ *     Run some tests: 1. Create an initial value, a copy and a fake cursor to use with the
```
Comment (fionaochee) [inline test/csuite/wt3338_partial_update/main.c:143]: Should the 1. start on a newline?

File: test/csuite/wt6185_modify_ts/main.c [hunk 6]
```diff
@@ -271,7 +268,7 @@ repeat(WT_SESSION *session, WT_CURSOR *c)
 }
 
 /*
- * reset --
+ * evict --
  *     Force eviction of the underlying page.
  */
 /*
```
Comment (fionaochee) [inline test/csuite/wt6185_modify_ts/main.c:274]: Should this TODO be removed?

File: test/csuite/wt2999_join_extractor/main.c (hunk 3 above)
Comment (jiechenbo) [reply test/csuite/wt2999_join_extractor/main.c:41]: Realised that the comment didn't describe the function but the whole test! Fixed it up

**Review by etienneptl — APPROVED** [review]
LGTM

File: test/csuite/wt3338_partial_update/main.c (hunk 5 above)
Comment (jiechenbo) [reply test/csuite/wt3338_partial_update/main.c]: Yeah, unfortunately this is clang format (automatic formatting tool). I have cleaned it up by separating through a whole line in between. Let me know if you thats better!

**Review by fionaochee — APPROVED** [review]
LGTM.

**Pull request merged**